repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -2948,7 +2957,7 @@ public void deleteClassification(String entityGuid, String classificationName) t
         AtlasPerfTracer.log(perf);
     }
 
-    private boolean classificationHasPendingTask(AtlasTask task, String classificationVertexId, String entityGuid) {
+    private boolean taskMatchingWithVertexIdAndEntityGuid(AtlasTask task, String classificationVertexId, String entityGuid) {
```

Comment:
Suggestion :- since method is returning boolean make sense to rename it to `isTaskMatchingWithVertexIdAndEntityGuid`

Reply:
addressed